app/javascript/src/components/Property/CautionaryAlerts.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+const CautionaryAlerts = ({ cautionaryAlerts }) => {
+
+	const isEmpty = (obj) => {
+		for(var prop in obj) {
+		  if(obj.hasOwnProperty(prop)) {
+			return false;
+		  }
+		}
+		return JSON.stringify(obj) === JSON.stringify({});
+	  }
+	
+	  let alertsToShow;
+	
+	  if (!isEmpty(cautionaryAlerts.alerts)) {
+		alertsToShow = <li className="govuk-tag bg-orange">Address Alert: {cautionaryAlerts.alerts.description} (<span className="govuk-!-font-weight-bold">{cautionaryAlerts.alerts.alertCode}</span>)</li>;
+	  } else {
+		alertsToShow;
+	  }
+
+	return (
+		<>
+			<ul className="hackney-property-alerts">
+              {alertsToShow}
+            </ul>
+		</>
+	)
+}
+
+export default CautionaryAlerts
```

Comment:
Can you change it to the 2space tab to be consistent with the other project, please?